src/redux/book/BookSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    AddBook: () => {},
+
+    RemoveBook: () => {},
```

Comment:
- [ ] As per these [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Includes%20a%20reducer%20that%20adds,reducer%20that%20removes%20a%20book), Please add the functionality of the `addBook` and `removeBook` reducers, not leaving them empty like that.
[Hint] It will be something like that:
```javascript
 reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
```

Reply:
I add state and action for both addBook and RemoveBook reducers